shoop/xtheme/parsing.py

```diff
@@ -171,6 +173,10 @@ class PlaceholderExtension(_PlaceholderManagingExtension):
     * The body of this block is actually discarded; only the inner `column`, `row` and `plugin`
       directives have any meaning.  (A parser-time `Layout` object is created and populated during parsing
       of this block.)
+    * An optional `global` parameter can be used to specify that the configuration for this placeholder
+      should be "global" across different views (although currently that only applies to placeholders within
+      the same template, i.e, if you defined the same global placeholder in different templates they will
+      not share configuration, but an included or base template that is rendered by different views will.
```

Comment:
Another point: Docstrings are rendered by Sphinx in our API documentation and should be valid ReST. The backticks are used incorrectly here (in your added parts and in the parts above it). Singly backticked items are references to a class, function, module etc. and Sphinx tries to make them links, but for things that are not references to any Python object, you should use doubly backticked text. For example:
```rst
...optional ``global`` parameter...
...inner ``column``, ``row`` and ``plugin`` directives...
...A parser-time `~shoop.xtheme.layout.Layout` object... (Note: Sphinx will render that as single
                                                          word "Layout" which will be a link to
                                                          documentation of  shoop.xtheme.layout.Layout)
... see also `LayoutPartExtension` ... (objects in same module are usually resolved
                                        automatically by Sphinx and don't need the full module path)
```

You should check how the documentation renders out by building the HTML documentation as instructed in our README: https://github.com/shoopio/shoop/#documentation (Generated documentation for this should appear to: `doc/_build/html/api/shoop.xtheme.html` under your Shoop checkout)